agent/agent-bootstrap/src/test/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/ConfigurationTest.java

```diff
@@ -176,6 +182,68 @@ public void shouldOverrideConnectionString() throws IOException {
         assertEquals("InstrumentationKey=11111111-1111-1111-1111-111111111111", configuration.connectionString);
     }
 
+    @Test
+    public void shouldOverrideRoleName() throws IOException {
+        envVars.set("APPLICATIONINSIGHTS_ROLE_NAME", "role name from env");
+        envVars.set("WEBSITE_SITE_NAME", "role name from website env");
+
+        Configuration configuration = loadConfiguration();
+        ConfigurationBuilder.overlayEnvVars(configuration);
+
+        assertEquals("role name from env", configuration.role.name);
```

Comment:
should it be configuration.role.instance?

Reply:
see test below for `configuration.role.instance` test

Reply:
changed 
```{
  "connectionString": "InstrumentationKey=...",
  "role": {
    "**name**": "my cloud role name"
  }
}
```

do we need to update the microsoft doc to reflect this change?

to 

```{
  "connectionString": "InstrumentationKey=...",
  "role": {
    "**instance**": "my cloud role name"
  }
}
```

Reply:
role `name` and role `instance` are two different fields

Reply:
i was confused.  instance field is not exposed in the json config file?

Reply:
it is exposed in json config file: https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#cloud-role-instance